src/components/Article.jsx

```diff
@@ -0,0 +1,46 @@
+import { useParams } from "react-router";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { convertDate } from "../utils.js";
+import { Loading } from "./Loading.jsx";
+
+export const Article = () => {
+  const { article_id } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [article, setArticle] = useState({});
+  useEffect(() => {
+    axios
+      .get(`https://news-api-40x5.onrender.com/api/articles/${article_id}`)
+      .then(({ data: { article } }) => {
+        setArticle(article);
+        setIsLoading(false);
+      });
+  }, [article_id]);
```

Comment:
Extracting this API call out to an exportable function would really clean up this useEffect 🧹 ✨ Plus has the added benefit of making the code for the API call reusable if needed in another component